force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1002,26 +990,26 @@ export default class GeFormRenderer extends LightningElement{
      * @param sectionsList - Array, all sections
      * @returns {boolean} - true if error message was generated, false if otherwise
      */
-    getDonorTypeValidationError(fieldWrapper, sectionsList) {
+    getDonorTypeValidationError(sectionsList) {
 
         // get data import record helper
-        const di_record = this.getDataImportHelper(fieldWrapper);
+        const dataImportHelper = this.getDataImportHelper();
 
         // donation donor validation depending on selection and field presence
-        let isError = (di_record.donationDonorValue === DONATION_DONOR.isAccount1) ?
-            di_record.isAccount1ImportedEmpty && di_record.isAccount1NameEmpty :
-            di_record.donationDonorValue === DONATION_DONOR.isContact1 &&
-            di_record.isContact1ImportedEmpty && di_record.isContact1LastNameEmpty;
+        let isError = (dataImportHelper.donationDonorValue === DONATION_DONOR.isAccount1) ?
+            dataImportHelper.isAccount1ImportedEmpty && dataImportHelper.isAccount1NameEmpty :
+            dataImportHelper.donationDonorValue === DONATION_DONOR.isContact1 &&
+            dataImportHelper.isContact1ImportedEmpty && dataImportHelper.isContact1LastNameEmpty;
```

Comment:
What do you think of breaking this up like so?
```suggestion
        const isAccountDonor = dataImportHelper.donationDonorValue === DONATION_DONOR.isAccount1;
        const areEmptyAccountFields = dataImportHelper.isAccount1ImportedEmpty && dataImportHelper.isAccount1NameEmpty;
        const isContactDonor = dataImportHelper.donationDonorValue === DONATION_DONOR.isContact1;
        const areEmptyContactFields = dataImportHelper.isContact1ImportedEmpty && dataImportHelper.isContact1LastNameEmpty;

        const isError = 
            isAccountDonor
            ? areEmptyAccountFields
            : isContactDonor && areEmptyContactFields;
```